portafly/doc/ux_specs/accounts/index.md

```diff
@@ -1,4 +1,4 @@
-# Audience > Accounts > Listing
+# Accounts index
```

Comment:
@cntlsn does this mean that for every page containing a list of elements (Accounts, Applications, Backends, etc.) we'll be using the term `index` ?

Reply:
You're correct @porueesq but let me elaborate a bit more.

An _index_ page serves as an index (in the strict sense) for a given collection. This index shows the list of all available accounts and the user can click and navigate to each one of them.

A page featuring a showcase of images, plain text items... wouldn't be considered an index.

In our case, I think it makes sense to assume all our _listings_ will be _indexes_. But let's make it clear they may not be.

Reply:
Although those pages are technically indexes I don't think we should use the term `index` in the UI, it's not very intuitive and could be redundant (ie. an index page with a list of Applications could just be called "Applications" instead of "Applications index").

We will still use the term `index` internally, and for technological reasons, to refer to those pages (eg. filenames)